src/models/classifier.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Tuple
+
+import numpy as np
+import pandas as pd
+
+from src.data.utils_sly import CLASS_MAP
+
+
+class EdemaClassifier:
+    """Hard-coded edema classifier."""
+
+    def __init__(self) -> None:
+        self._output: List = []
+
+    def classify(
+        self,
+        df_in: pd.DataFrame,
+    ) -> pd.DataFrame:
+        """The main classification function.
+
+        Args:
+            df: initial dataframe containing images' IDs and features.
+
+        Returns:
+            DataFrame with identified edema severity class.
+        """
+        # TODO: ensure that EdemaClassifier can process empty dataframes
+        img_groups = df_in.groupby('Image name')
+        for image_name, df_img in img_groups:
+            # FIXME: Extraction of feature and its confidence is not necessary
+            # FIXME: I think it is enough if you add two columns (Class ID and Class)
+            edema_severity, feature, confidence = EdemaClassifier._get_edema_severity(df_img)
```

Comment:
@mak-en 

1. Make sure that we may process empty data frames.
2. Addition of class ID and class should be enough.